### ISSUE TITLE
Install Odoo 12 Enterprise w/Docker

### ISSUE THREAD

**Issue by aev-treming** [odoo/docker#291]
What can I do if I want to install Odoo Enterprise using Docker?

**Comment by Sirfanas** [comment]
Hi,
You have to get Enterprise addons on your server, then you have to bind a volume mount point on your container so it see this folder, and add it in conf files.

Here's is the step:
I'll consider you are working under /opt/odoo (Just adapt to your system config)

```
git clone git@github.com:odoo/enterprise.git -b 12.0
# Change 12.0 by the version you need
```

If you use docker run / create command:
`
docker run/create [all your parameter] --volume=/opt/odoo/enterprise:/mnt/enterprise odoo:12.0
`

If you use a docker-compose file:
```
services:
    odoo:
        image: odoo:12.0
        volume:
             - ./enterprise:/mnt/enterprise
...
```

Then in your odoo.conf you have to edit addons_path and add /mnt/enterprise

Restart your container and create a new database, it should be in enterprise !

**Comment by d-fence** [comment]
Thanks for the answer

**Comment by Christian4Mac** [comment]
Apologies if I should have created a new issue on this, but I'm having trouble getting the above solution to work for me.

I have the `enterprise:13.0` apps mapped to `/mnt/enterprise` in the `odoo:13` container, and set addons_path in odoo.conf: `addons_path = /mnt/enterprise, /mnt/extra-addons`

It appears that no matter what I do, the community addons seem to load before the enterprise ones.
Odoo runs the community edition, but the enterprise apps still show up (field service, amazon, etc.), so I believe I've at least configured the directories, network, and db properly?

```
2020-02-18 04:35:00,392 1 INFO ? odoo: Odoo version 13.0-20200121 
2020-02-18 04:35:00,392 1 INFO ? odoo: Using configuration file at /etc/odoo/odoo.conf 
2020-02-18 04:35:00,392 1 INFO ? odoo: addons paths: ['/usr/lib/python3/dist-packages/odoo/addons', '/var/lib/odoo/addons/13.0', '/mnt/enterprise', '/mnt/extra-addons'] 
```

I've tried reordering the four addon paths to force `/mnt/enterprise` to load first. e.g.
`addons_path = /mnt/enterprise, /usr/lib/python3/dist-packages/odoo/addons, /var/lib/odoo/addons/13.0, /mnt/extra-addons`
I've also attempted this through odoo command-line, and resetting the database, to no avail

Maybe I'm just missing something simple, or I'm going about this the wrong way? I remember having to pay attention to the addon order when installing from source, but this seemed like odd behavior to me.
